app.js

```diff
@@ -1,13 +1,21 @@
+// Imports
 const express = require('express');
 const mongoose = require('mongoose');
 const cors = require('cors');
+const { celebrate, Joi, errors } = require('celebrate');
+const { requestLogger, errorLogger } = require('./middlewares/logger');
+require('dotenv').config();
 
-// Models
+const { login, createUser } = require('./controllers/users');
+const auth = require('./middlewares/auth');
+
+// Routers
 const workersRouter = require('./routes/workers');
+const usersRouter = require('./routes/users');
 
+// Rutas & middleware
 const app = express();
 const { PORT = 3000 } = process.env;
-
 app.use(express.json());
 app.use(cors());
 app.options('*', cors());
```

Comment:
**This is a good way to develop an `API`.** 🥷🏼

When you have a project in production keep in mind to list the clients that will be connected to this server.